library/include/modules/Units.h

```diff
@@ -205,5 +205,13 @@ DFHACK_EXPORT extern const std::vector<int32_t> stress_cutoffs;
 DFHACK_EXPORT int getStressCategory(df::unit *unit);
 DFHACK_EXPORT int getStressCategoryRaw(int32_t stress_level);
 
+enum ActionTypeGroup {All, Movement, MovementFeet, Offensive, Work};
```

Comment:
a good practice for any enum is to make the first element invalid so unset enums are easily identified.

Reply:
How about just "None"?

Reply:
It's better to make it explicitly "wrong", like "Unset" or "Invalid". Then you can print a WARNING() and plugin authors have a chance of discovering their error instead of the call failing silently. Naming it "None" implies that it's a valid value and shouldn't generate a warning.

Reply:
you can also overrtide the default 0 for the first element by using `INVALID = -1` as the first entry, which will make the second entry 0 

also, in _most_ of our code base enum constants are defined in ALL CAPS, although there are exceptions to this and i don't think we have a formal style statement on this anywhere